src/lib.rs

```diff
@@ -548,9 +513,20 @@ decl_storage! {
         /// Post identifier value to be used for for next post created.
         pub NextPostId get(next_post_id) config(): PostId;
 
+        /// Max depth of category.
+        pub MaxCategoryDepth get(max_category_depth) config(): u8;
+
         /// Account of forum sudo.
         pub ForumSudo get(forum_sudo) config(): Option<T::AccountId>;
 
+        /// Moderator set for each Category
+        pub CategoryByModerator get(category_by_moderator) config(): double_map CategoryId, blake2_256(T::AccountId) => bool;
+
+        pub ReactionByPost get(reaction_by_post) config(): double_map PostId, blake2_256(T::AccountId) => PostReaction;
```

Comment:
Like in `CategoryByModerator` you are here identifying a forum member by account.

Reply:
it will be updated after solution of unified Identity.